### PR TITLE
fix: definition/term roles

### PR DESCRIPTION
**Pull request by marcosvega91**
Regarding the issue on [`@testing-library/dom`](https://github.com/testing-library/dom-testing-library/issues/703), I have noticed that `term` and `definition` were not correct

### CODE REVIEW

File: src/etc/roles/literal/definitionRole.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ const definitionRole: ARIARoleDefinition = {
   prohibitedProps: [],
   props: {},
   relatedConcepts: [
+    {
+      concept: {
+        name: 'dfn',
```
Comment (eps1lon) [inline src/etc/roles/literal/definitionRole.js:17]: [`dfn` does not map to `definition`](https://www.w3.org/TR/html-aam-1.0/#details-id-29).

File: src/etc/roles/literal/termRole.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const termRole: ARIARoleDefinition = {
   relatedConcepts: [
     {
       concept: {
-        name: 'dfn',
+        name: 'dt',
```
Comment (eps1lon) [inline src/etc/roles/literal/termRole.js:17]: `dfn` **and** `dt` map to `term`: 
- [`dfn` mapping](https://www.w3.org/TR/html-aam-1.0/#details-id-29)
- [`dt` mapping](https://www.w3.org/TR/html-aam-1.0/#details-id-33)

**Comment by eps1lon** [comment]
Also: The source of truth for these mappings is `scripts/roles.json` from which the individual files are created via `node scripts/breakUpAriaJSON.js`

File: src/etc/roles/literal/termRole.js (hunk 2 above)
Comment (marcosvega91) [reply src/etc/roles/literal/termRole.js:17]: I didn't look for `dfn` on specification but reading the `README.md` I thought was correct the `definition` role

`'{"name": "dfn"}' => Set { 'definition' },`

Comment (eps1lon) [reply src/etc/roles/literal/termRole.js:17]: > but reading the README.md

Right, we should update that one as well

**Comment by eps1lon** [comment]
@marcosvega91 Much appreciated, thanks!

**Comment by AriPerkkio** [comment]
Sorry to comment on this old closed issue but would it be possible to release fix version including this PR? Last release seems to be 8 months old.

I keep finding myself adding `// TODO: Replace with getByRole('term') once aria-query#70 is released` comments on multiple projects.

**Pull request merged**